variants/aws-ecs-1-nvidia/Cargo.toml

```diff
@@ -20,6 +20,7 @@ included-packages = [
 # core
     "release",
     "kernel-5.10",
+    "wicked",
```

Comment:
The wicked variants should also add a dependency on wicked in the `build-dependencies` section below.